README.md

```diff
@@ -33,7 +33,8 @@ jobs:
 
       - name: Link Checker
         id: lychee
-        uses: lycheeverse/lychee-action@v1.5.0
+        # If you always want to use the latest features, you can replace the version lycheeverse/lychee-action@master
+        uses: lycheeverse/lychee-action@v1.5.1
```

Comment:
Not so sure about this comment. In the Readme it's explicitly advertised to pin versions and even use hashes: https://github.com/lycheeverse/lychee-action#security-and-updates
Using `@master` can also lead to unexpected breaking of the action.

Supporting "version branches" like other actions do, would at least prevent breaking changes: 
`@v1` (aka "give me the newest v1 minor/patch available with all new features and fixes without breaking changes")

Reply:
This is a good suggestion, but I tested the v1 version and found that the operation failed. I haven't found the reason yet.

Reply:
I think that needs to be enabled and supported specifically.
Which is not the case yet.

Reply:
Guess we have to push the v1 tag whenever we do a new release. We can do that at later point in time. For now I agree with @tooomm; let's remove the comment for the reasons he mentioned.

Reply:
Agree

Reply:
> Guess we have to push the v1 tag whenever we do a new release.

https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-release-management-for-actions

Reply:
Thanks for the link.